docs/src/test/java/docs/http/javadsl/server/directives/TimeoutDirectivesExamplesTest.java

```diff
@@ -177,4 +177,31 @@ public void testRequestTimeoutCustomResponseCanBeAddedSeparately() {
         assert (StatusCodes.ENHANCE_YOUR_CALM.equals(statusCode));
         //#withRequestTimeoutResponse
     }
+
+    @Test
+    public void extractRequestTimeout(){
+        //#extractRequestTimeout
+        Duration timeout1 = Duration.create(500, TimeUnit.MILLISECONDS);
+        Duration timeout2 = Duration.create(1000, TimeUnit.MILLISECONDS);
+        Route route =
```

Comment:
We use 2 spaces as indentation for Java doc snippets as well for better readability.